statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1116,11 +1116,17 @@ def test_conf_int(self):
         assert_almost_equal(self.res1.conf_int(), self.res2.conf_int,
                             DECIMAL_2)
 
+    @pytest.mark.xfail(reason="No idea!  But until this test is actually "
+                              "written, it is considered broken.",
+                       strict=True, raises=NotImplementedError)
     def test_predict(self):
-        pass
+        raise NotImplementedError
 
+    @pytest.mark.xfail(reason="No idea!  But until this test is actually "
```

Comment:
Could you use a simpler reason, e.g., "test has not been implemented for this class"?  A one-line reason seems to be enough.  Also, would be great to use the same simple reason everywhere so these are coalesced in pytest reports.

Reply:
Sure.  FWIW the "No idea!" is in place for several similar xfails, so you can search on that if you're looking for where else this has been done.